<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GridsquareForm</class>
 <widget class="QWidget" name="GridsquareForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListView" name="gridsquare_list"/>
      <widget class="QListWidget" name="result_labels"/>
     </widget>
     <widget class="ImageViewer" name="gridsquare_view"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>imageviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>gridsquare_list</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>GridsquareForm</receiver>
   <slot>loadGridsquare()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadGridsquare()</slot>
 </slots>
</ui>
